trieste/acquisition/function/multi_objective/dominance.py

```diff
@@ -17,7 +17,7 @@
 
 import tensorflow as tf
 
-from ...types import TensorType
+from ....types import TensorType
```

Comment:
I don't understand why you moved `acquisition.multi_objective` inside `acquisition.function`. AFAIUI:

* `acqusition` is everything connected to acquisition functions, including interfaces, rules, samplers, optimizers, etc.
* `acquisition.function` is for _specific_ acquisition function implementations
* `acqusition.multi_objective` is for multi-objective helper functions

Reply:
multi objective helper functions in there are really in service of multi BO acq functions, like in hypervolume.py, so it made sense to move them there